mirrormanager2/templates/fedora/_macros.html

```diff
@@ -280,3 +280,34 @@ <h4>Export Compliance</h4>
     {{ form.csrf_token }}
   </form>
     {% endmacro %}
+
+
+    {% macro pagination_bar(result) %}
+    {% if result.total_pages > 1 %}
+      <nav aria-label="Pagination">
+        <ul class="pagination pagination-sm justify-content-center my-4">
+          {# Page list #}
+          {% for page_number in result.truncated_pages_list(margin=6) %}
+            {% if page_number == result.page_number %}
+              <li class="page-item active" aria-current="page">
+                <span class="page-link">
+                  {{ page_number }}
+                  <span class="sr-only">(current)</span>
+                </span>
+              </li>
+            {% elif page_number == None %}
+            <li class="page-item disabled">
+              <span class="page-link">...</span>
+            </li>
+            {% else %}
+              <li class="page-item">
+                <a class="page-link" href="{{ result.page_url(page_number) }}">
+                  {{ page_number }}
+                </a>
+              </li>
+            {% endif %}
+          {% endfor %}
+        </ul>
+      </nav>
+    {% endif %}
+  {% endmacro %}
```

Comment:
I think the indentation is wrong on this one, the `macro` and the `if` were on the same level at the start.

Reply:
fixed this issue here, but that whole file is kinda indented wacky to begin with -- i will fix this in a differeent PR